main.js

```diff
@@ -39,6 +40,35 @@ process.on('unhandledRejection', (err) => {
   logger.error(err);
 });
 
+const twitterTimeout = process.env.TWITTER_TIMEOUT || 60000;
+const twitterCache = new TwitterCache({
+  clientInfo:
+  {
+    consumer_key: process.env.TWITTER_KEY,
+    consumer_secret: process.env.TWITTER_SECRET,
+    access_token_key: process.env.TWITTER_ACCESS_TOKEN_KEY,
+    access_token_secret: process.env.TWITTER_ACCESS_TOKEN_SECRET,
+  },
+  toWatch:
+  [
+    { acc_name: '@playwarframe', plain: 'warframe' },
```

Comment:
could this be in a resource json that we can add to without having to edit source?